msg/messages.js

```diff
@@ -239,6 +239,9 @@ Blockly.Msg.CONTROLS_IF_ELSE_TITLE_ELSE = Blockly.Msg.CONTROLS_IF_MSG_ELSE;
 /// tooltip - Describes the 'else' subblock during [https://github.com/google/blockly/wiki/IfElse#block-modification if block modification].
 Blockly.Msg.CONTROLS_IF_ELSE_TOOLTIP = 'Add a final, catch-all condition to the if block.';
 
+/// block text - See [https://github.com/google/blockly/wiki/IfElse https://github.com/google/blockly/wiki/IfElse].  The English word "otherwise" would probably be superior to "else", but the latter is used because it is traditional and shorter.
+Blockly.Msg.CONTROLS_IFELSE_TITLE = 'if %1 do %2 else %3';
```

Comment:
@AnmAtAnm Thanks for edit.  It just occurred to me that we already have all the words for the translations of this edit.  A quick Python script would be able to merge together the three existing messages in each language and inject the new message at the right location.

* Saves time for the translators.
* Improves consistency.
* Saves time for us due to not needing to deal with some message that has "% 2" instead of "%2" or has some other error.